frontend/test/ui/pages/desktop/home.py

```diff
@@ -0,0 +1,21 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Home(Base):
+
+    @property
+    def header(self):
+        return self.Header(self)
+
+    class Header(Region):
+
+        _root_locator = (By.CLASS_NAME, 'banner')
+
+        _copter_locator = (By.CLASS_NAME, 'copter')
```

Comment:
So this is for the header of the main home page. I guess since this is a Single Page Application, until you click on an experiment I didn't know if it should be called base, or home. But the ```copter``` is still something that should be checked as displayed in my opinion.